mozetl/hardware_report/summarize_json.py

```diff
@@ -0,0 +1,691 @@
+"""
+This job was originally located at [1].
+
+[1] https://git.io/vHRWO
+"""
+import ujson as json
+import datetime as dt
+import os.path
+import boto3
+import botocore
+import requests
+import moztelemetry.standards as moz_std
+from pyspark.sql import SQLContext
+from pyspark.sql import SparkSession
+
+# Reasons why the data for a client can be discarded.
+REASON_INACTIVE = "inactive"
+REASON_BROKEN_DATA = "broken"
+# These fields have a fixed set of values and we need to report all of
+# them.
+EXCLUSION_LIST = ["has_flash", "browser_arch", "os_arch"]
+
+S3_PUBLIC_BUCKET = "telemetry-public-analysis-2"
```

Comment:
Let's actually have a `bucket` param in the cli instead of using a static `S3_PUBLIC_BUCKET` - make it required. This is pretty standard for our airflow jobs. It will also allow us to test this etl job on the non-prod bucket before deploying.